### PR TITLE
Update ac.ir

### PR DESCRIPTION
**Pull request by technobulb**
Added Model: Danby DAC060EB7WDB

# What's new

Added Model: Danby DAC060EB7WDB
Sold at costco

# Checklist (For Reviewer)

-I am new to coding so please let me know if there is anything needing correction

### CODE REVIEW

File: assets/resources/infrared/assets/ac.ir [hunk 1]
```diff
@@ -408,3 +408,40 @@ type: raw
...
+# 
+name: Cool_lo
+type: raw
+frequency: 38000
+duty_cycle: 0.330000
+data: 4401 4441 528 1629 529 550 528 1628 529 551 528 550 528 551 527 551 528 1629 529 1629 529 550 529 1630 528 551 528 549 530 550 529 551 528 549 529 550 529 1629 529 1628 530 549 530 1629 529 550 529 1628 529 1629 529 1631 527 1628 530 1628 529 1629 528 1628 530 1629 529 1629 529 1629 529 1629 528 1629 529 1629 529 1630 528 1629 529 1629 529 1628 529 1629 528 551 528 1629 529 550 529 550 530 548 529 1631 527 551 528 1629 529 5235 4402 4439 530 550 528 1629 529 549 530 1628 529 1629 529 1628 530 1629 529 549 530 550 529 1628 530 552 526 1628 529 1628 530 1628 530 1627 531 1628 529 1629 528 551 528 550 529 1628 530 550 528 1628 529 549 529 550 528 550 529 549 530 548 530 551 528 550 528 578 500 550 529 550 529 551 527 549 530 549 529 549 529 550 528 548 530 550 528 549 529 1629 528 550 529 1630 528 1628 530 1628 530 549 530 1628 529 549 529
+# 
+name: Fan_hi
```
Comment (gsurkov) [inline assets/resources/infrared/assets/ac.ir]: There are no `Fan_hi` and `Fan_lo` signals names in the file format. 
Please see the [documentation](https://github.com/flipperdevices/flipperzero-firmware/blob/dev/documentation/UniversalRemotes.md#air-conditioners) for available signal names.

**Review by gsurkov — CHANGES_REQUESTED** [review]

File: assets/resources/infrared/assets/ac.ir (hunk 1 above)
Comment (technobulb) [reply assets/resources/infrared/assets/ac.ir]: Ok I fixed. Sorry didnt realize we were limited to those, thought that was the minimum

**Review by technobulb — COMMENTED** [review]
Removed Fan

**Review by gsurkov — APPROVED** [review]

**Review by skotopes — APPROVED** [review]

**Comment by technobulb** [comment]
Thanks. It seems like you add this to the universal codes. How can I add this to my flipper in the universal file instead of standalone preset? Do you add it in the next software update?

**Pull request merged**